02-peer-protocol.md

```diff
@@ -70,6 +70,9 @@ If this fails at any stage, or a node decides that the channel terms
 offered by the other node are not suitable, the channel establishment
 fails.
 
+Note that multiple channels can operate in parallel, but only after
```

Comment:
Excellent. IMO the allowance of parallel channels was made implicit by using chan ID's (before this PR), but as always it's better to make things explicit when possible 😎